data/coalition/coalition missions.txt

```diff
@@ -1866,3 +1866,469 @@ mission "Saryd University Lecture"
+	to complete
+		never
+	to fail
+		has "Coalition: Expeditions Past 9: offered"
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Sedlitaris hasn't entered the system yet. Better depart and wait for it.`
+
+mission "Coalition: Expeditions Past 9"
+	landing
+	name "Escape Route"
+	description "Travel southwest of <system>, on a route to the Ablub system, so that Sedlitaris may look for a signal frequency of the expedition."
+	passengers 1
+	to offer
+		has "Coalition: Expeditions Past 8: active"
+	source "Hopper"
+	on offer
+		conversation
+			`You fly around <origin> as Sedlitaris searches for the pylon's exact position, in the hopes that she can bring it back as well. She tells you to land by a mountain range, and has you readjust your landing spot every so often as she tries to nail the exact location of the pylon. "Buried it to protect it from the atmosphere's acidity, they must have," she comments.`
+			`	By the time she's done, you open your ship's hatch right to where the pylon should be and help her dig it out, taking care to not be out for too long. When the device is dug out, you bring it inside your ship while Sedlitaris fills up the hole with what dirt had been dug out.`
+			`	As you're flying to the hangars, she plays the pylon's entry: "Log started. Made a route from Summit straight to the meeting point, I had, but find the split group, I could not. Restless, agitated, the crew has become, heading straight to Ablub, we are, after buried, this thing is. If find this, anyone from the split group is, head back-" at that point an untranslated shouting muffles what Mimodum was saying, followed by dozens of legs in a rush. "Oh no..." is heard right before the recording ends.`
+			`	You focus on flying the ship, but even when you've landed, Sedlitaris remains silent.`
+			choice
+				`	"Are you alright?"`
+				`	"He said they were going back immediately, but they never made it. You think that was the last pylon?"`
+				`	(Stay quiet).`
+			`	She goes without a word for a while still. "Set a route for Ablub, could you, Captain? If follow the same route they used, we do, and if any more to find, there is, pick up the signals, I will."`
+			`	You agree, and look to set a route straight to Ablub.`
+				accept
+	on enter "Girtab"
+		dialog `While Sedlitaris remained in her bunk since you left for Ablub, you can hear her shouting from her bunk as you enter the system. She comes running to you, bringing along the pylon. "Picked up a signal, it has!" She says, pointing at the machine's panel, where some message is repeatedly beeping in and out. "Land on that planet we must, Captain!" You tell her you'll land soon, and she remains with you to figure out the pylon's location on Harmony, once you've landed.`
```

Comment:
```suggestion
		dialog `While Sedlitaris remained in her bunk since you left for Ablub, you can hear her shouting from her bunk as you enter the system. She comes running to you, bringing along the pylon. "Picked up a signal, it has!" She says, pointing at the machine's panel, where some message is repeatedly beeping in and out. "Land on that planet we must, Captain!" You tell her you'll land soon, and she remains with you to figure out the pylon's location on Harmony once you've landed.`
```